### PR TITLE
GH Python docstrings

### PR DESCRIPTION
**Pull request by tomvanmele**
### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/compas_ghpython/artists/__init__.py [hunk 1]
```diff
@@ -5,6 +5,12 @@
 
 .. currentmodule:: compas_ghpython.artists
 
+.. rst-class:: lead
+
+Artists for visualizing (painting) COMPAS geometry, robots, and data structures in Grasshopper using the GH Python component.
+The artists in this package provide plugins for the pluggable methods of the base artists in :mod:`compas.artists`.
+Therefire, they can be used directly, from here, or through the base artists :class:`compas.artists.Artist`.
```
Comment (gonzalocasas) [inline src/compas_ghpython/artists/__init__.py]:
```suggestion
Therefore, they can be used directly, from here, or through the base artists :class:`compas.artists.Artist`.
```

File: src/compas_ghpython/artists/robotmodelartist.py [hunk 2]
```diff
@@ -36,5 +43,13 @@ def create_geometry(self, geometry, name=None, color=None):
...
+    # should this not have a return value?!
     def draw(self):
+        """Draw the robot model.
+
+        Returns
+        -------
+        None
+
+        """
         self.draw_visual()
```
Comment (gonzalocasas) [inline src/compas_ghpython/artists/robotmodelartist.py]: yes, this is a bug. it should return `draw_visual()`  which is a generator of rhino meshes. 
```suggestion
    def draw(self):
        """Draw the visual meshes of the robot model.

        Returns
        -------
        generator[:rhino:`Rhino.Geometry.Mesh`]

        """
        return self.draw_visual()
```

And btw, the `draw_*` methods in the class `compas.artists.RobotModelArtist` are incorrectly documented as returning `None`, when most of them yield rhino meshes.

Comment (gonzalocasas) [reply src/compas_ghpython/artists/robotmodelartist.py]: and 2nd btw: is there a convention for how to write generators in the returns? or should it be changed to `sequence[]` instead?

**Review by gonzalocasas — APPROVED** [review]
LGTM + minor details

File: src/compas_ghpython/artists/robotmodelartist.py (hunk 2 above)
Comment (tomvanmele) [reply src/compas_ghpython/artists/robotmodelartist.py]: if it yields something it should be a generator, if it an ordered random access collection it should be a sequence. the proper type hint would be `Generator[...]` and `Sequence[...]`, but in my opinion lowercase is fine as well in docstrings

Comment (gonzalocasas) [reply src/compas_ghpython/artists/robotmodelartist.py]: 👍  (yep, it's a generator)

**Pull request merged**